src/callbacks_step/amr_dg3d.jl

```diff
@@ -250,7 +250,7 @@ function coarsen!(u_ode::AbstractVector, adaptor,
                 # If an element is to be removed, sanity check if the following elements
                 # are also marked - otherwise there would be an error in the way the
                 # cells/elements are sorted
-                @assert all(to_be_removed[old_element_id:(old_element_id + 2^ndims(mesh) - 1)]) "bad cell/element order"
+                @assert all(to_be_removed[old_element_id:(old_element_id + 2^3 - 1)]) "bad cell/element order"
```

Comment:
# Increment `element_id` on the coarsened mesh by one and `skip` = 7 in 2D
                # because 4 children elements become 1 parent element